lib/sidekiq/web_helpers.rb

```diff
@@ -166,6 +166,7 @@ def truncate(text, truncate_after_chars = 2000)
     end
 
     def display_args(args, truncate_after_chars = 2000)
+      args = [args] unless args.respond_to?(:map)
```

Comment:
may be is better to use `Array()` instead conditional? I.e. `Array(args).map {}`